pkg/linter/linterTypes.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2023- IBM Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package linter
+
+import "github.com/np-guard/vpc-network-config-analyzer/pkg/vpcmodel"
+
+type linter interface {
+	check() ([]string, error) // returns []string of size 0 if no lint issues
+	getName() string
+	getFindings() []any
```

Comment:
sounds right. Just one comment: this implies a very local context in each lint msg. Namely, each lint msg will contain the vpc name in addition to the other details - instead of having a list of all linting comments of each vpc

Reply:
done